src/diagnostics/riscv64/disasm-riscv64.cc

```diff
@@ -223,6 +228,11 @@ void Decoder::PrintShamt32(Instruction* instr) {
   out_buffer_pos_ += SNPrintF(out_buffer_ + out_buffer_pos_, "%d", imm);
 }
 
+void Decoder::PrintRvcImm6(Instruction* instr) {
```

Comment:
Add `PrintRvcShamt()` mimicing `PrintShamt()`. It seems that extraction of shift amount immediate value is handled separately from other `Imm6` values. Let's just follow the convention of the regular RISCV instructions. You may need to add sth like `instr->RvcShamt()` too

Reply:
These functions are ready and will come with other CI type instructions in these days.